templates/event.php

```diff
@@ -22,38 +22,26 @@
 gp_tmpl_load( 'events-header', get_defined_vars(), __DIR__ );
 ?>
 
-<div class="event-details-page">
+<div class="event-page-wrapper">
 	<div class="event-details-head">
 		<h1>
 			<?php echo esc_html( $event_title ); ?>
 			<?php if ( 'draft' === $event->post_status ) : ?>
 				<span class="event-label-draft"><?php echo esc_html( $event->post_status ); ?></span>
 			<?php endif; ?>
 		</h1>
-		<p>Host: <a href="<?php echo esc_attr( get_author_posts_url( $event->post_author ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $event->post_author ) ); ?></a></p>
+		<p>
+			Host: <a href="<?php echo esc_attr( get_author_posts_url( $event->post_author ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $event->post_author ) ); ?></a>
+			<?php if ( current_user_can( 'edit_post', $event_id ) ) : ?>
+				<a class="event-page-edit-link button" href="<?php echo esc_url( gp_url( 'events/edit/' . $event_id ) ); ?>"><span class="dashicons dashicons-edit"></span>Edit event</a>
```

Comment:
For me the edit button doesn't show the pencil icon:

<img width="173" alt="Screenshot 2024-02-20 at 11 52 36" src="https://github.com/WordPress/wporg-gp-translation-events/assets/550401/e0d0bb19-3111-454b-bdc8-4a448e8c22f1">

Reply:
The icon is still not showing but I couldn't figure out why. I tested on firefox and chrome and it doesn't show in either.